src/models/GasRefundParticipation.ts

```diff
@@ -74,16 +74,20 @@ export class GasRefundParticipation extends Model<EpochGasRefundData> {
   accumulatedGasUsedChainCurrency: string; // @debug
 
   @Column(DataType.DECIMAL)
-  accumulatedGasUsedPSP: string;
+  accumulatedGasUsedPSP: string; // @debug
```

Comment:
btw, what does [`@debug`](https://github.com/debug) do?

Reply:
Just a dev hint to say column isn't used in any business logic.
But if you're question is about jsdoc,  I'm out